### PR TITLE
Support new Unix primitive names in lwt

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

**Comment by dra27** [comment]
(the previous version of the package isn't needed anymore since Dune now does `bigarray` "correctly")

File: packages/lwt/lwt.5.5.2~alpha-repo/opam [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  "Raphaël Proust <code@bnwr.net>"
+  "Anton Bachin <antonbachin@yahoo.com>"
+]
+dev-repo: "git+https://github.com/ocsigen/lwt.git"
+
+depends: [
+  "cppo" {build & >= "1.1.0"}
+  "dune" {>= "1.8.0"}
+  "dune-configurator"
+  "bigarray-compat"
```
Comment (kit-ty-kate) [inline packages/lwt/lwt.5.5.2~alpha-repo/opam:23]: Is it really still needed?
```suggestion
  "bigarray-compat"
```

Comment (dra27) [reply packages/lwt/lwt.5.5.2~alpha-repo/opam:23]: Oh no - I just copied it from yours 😁

**Comment by kit-ty-kate** [comment]
Thanks! Mmh, actually could you also move it to `lwt.5.6.0.1~alpha-repo` for this one and also remove the previous one?

The reason for the version number is that the next version is going to be `lwt.5.6.0` because of the removal of the `result` dependency and some packages in opam-repository already anticipates this by constraining against this new version.

**Comment by dra27** [comment]
Done!

**Pull request merged**